src/saveload/afterload.cpp

```diff
@@ -3263,6 +3263,12 @@ bool AfterLoadGame()
 		ScriptObject::InitializeRandomizers();
 	}
 
+	if (IsSavegameVersionBefore(SLV_COMPANY_INAUGURATED_PERIOD)) {
+		for (Company *c : Company::Iterate()) {
+			c->inaugurated_year_calendar = 1950;
```

Comment:
We do have a nice constant for the original start year of 1920: `CalendarTime::ORIGINAL_BASE_YEAR`.

That said, I'd suggest setting it to the starting year of this game, which is generally accurate for single-player games (the most likely type of older save to get loaded). 🙂 

```suggestion
			c->inaugurated_year_calendar = _settings_game.game_creation.starting_year;
```